### ISSUE TITLE
TRAIN always make putty inactive: software caused network error

### ISSUE THREAD

**Issue by taurenshaman** [lengstrom/fast-style-transfer#85]
Environment:  
Azure VM NV6:  
GPU Nvidia Tesla M60 x1  

> root@mosp: /mnt/mosp$ python style.py --style styles/starry-night-640x400.jpg --checkpoint-dir checkpoints --checkpoint-iterations 500 --batch-size 10
I tensorflow/stream_executor/dso_loader.cc:135] successfully opened CUDA library libcublas.so.8.0 locally
I tensorflow/stream_executor/dso_loader.cc:135] successfully opened CUDA library libcudnn.so.5 locally
I tensorflow/stream_executor/dso_loader.cc:135] successfully opened CUDA library libcufft.so.8.0 locally
I tensorflow/stream_executor/dso_loader.cc:135] successfully opened CUDA library libcuda.so.1 locally
I tensorflow/stream_executor/dso_loader.cc:135] successfully opened CUDA library libcurand.so.8.0 locally
Train set has been trimmed slightly..
(1, 400, 640, 3)
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE3 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.1 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
I tensorflow/core/common_runtime/gpu/gpu_device.cc:885] Found device 0 with properties:
name: Tesla M60
major: 5 minor: 2 memoryClockRate (GHz) 1.1775
pciBusID 97cf:00:00.0
Total memory: 7.93GiB
Free memory: 7.86GiB
I tensorflow/core/common_runtime/gpu/gpu_device.cc:906] DMA: 0
I tensorflow/core/common_runtime/gpu/gpu_device.cc:916] 0:   Y
I tensorflow/core/common_runtime/gpu/gpu_device.cc:975] Creating TensorFlow device (/gpu:0) -> (device: 0, name: Tesla M60, pci bus id: 97cf:00:00.0)
I tensorflow/core/common_runtime/gpu/gpu_device.cc:975] Creating TensorFlow device (/gpu:0) -> (device: 0, name: Tesla M60, pci bus id: 97cf:00:00.0)
UID: 25
Epoch 0, Iteration: 500, Loss: 1.43739e+07
style: 6.15022e+06, content:6.91703e+06, tv: 1.30661e+06
Epoch 0, Iteration: 1000, Loss: 1.21609e+07
style: 4.72436e+06, content:6.58813e+06, tv: 848353.0
Epoch 0, Iteration: 1500, Loss: 1.1245e+07
style: 4.08106e+06, content:6.39631e+06, tv: 767614.0
Epoch 0, Iteration: 2000, Loss: 1.04428e+07
style: 3.53381e+06, content:6.19862e+06, tv: 710418.0
Epoch 0, Iteration: 2500, Loss: 1.02348e+07
style: 3.60832e+06, content:5.9756e+06, tv: 650848.0
Epoch 0, Iteration: 3000, Loss: 9.48118e+06
style: 2.95307e+06, content:5.91172e+06, tv: 616389.0
Epoch 0, Iteration: 3500, Loss: 9.72546e+06
style: 3.29472e+06, content:5.84574e+06, tv: 584993.0
Epoch 0, Iteration: 4000, Loss: 9.24364e+06
style: 2.87614e+06, content:5.81101e+06, tv: 556496.0
Epoch 0, Iteration: 4500, Loss: 9.06416e+06
style: 2.61211e+06, content:5.90668e+06, tv: 545369.0

I tried 3 times and got an estimate: it will take about 30 minutes every 500 iterations.  
But, everytime about 30 minutes after Iteration: 4500, putty will turn to inactive, with a message: software caused network error.  
putty settings:  

- try 1: putty connection: 0 seconds to turn off (default)  
- try 2: putty connection: 30 seconds to turn off (default)  
- try 3: putty connection: 600 seconds to turn off (default)  

Thank you very much!
